src/main/java/ReadFile.java

```diff
@@ -0,0 +1,59 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ReadFile {
```

Comment:
I feel the class name shouldn't sound imperative. Perhaps something like `DukeFileReader` (to prevent confusion with `FileReader`) would be better?